packages/dev/loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -205,10 +206,13 @@ export class GLTFLoader implements IGLTFLoader {
     private _uniqueRootUrl: Nullable<string> = null;
     private _gltf: IGLTF;
     private _bin: Nullable<IDataBuffer> = null;
-    private _babylonScene: Scene;
+    /** @internal */
+    public _babylonScene: Scene;

```

Comment:
There is a public getter for this. Does that work?

Reply:
Yup it does! I hadn't found it yet